script/mobilenets-tensorflow/benchmark.py

```diff
@@ -239,7 +278,7 @@ def do(i, arg):
             rho=int(r['dict']['env']['CK_ENV_TENSORFLOW_MODEL_MOBILENET_RESOLUTION'])
 
             record_repo='local'
-            record_uoa='mobilenets-'+experiment_type+'-'+str(rho)+'-'+str(alpha)+'-'+lib_tags
+            record_uoa='mobilenets-'+experiment_type+'-'+str(rho)+'-'+str(alpha)+'-'+lib_tags+'-'+program_id
```

Comment:
With TFLite, the repo name is somewhat awkward:
```
mobilenets-performance-128-0.25-tensorflow-0.1.7-tflite
```
How about we simply change `lib_tags` from:
```
lib_tags='tensorflow-'+r['dict']['customize']['version']
```
to
```
lib_tags=program_id+'-'+r['dict']['customize']['version']
```
This should result in repo names like:
```
mobilenets-performance-128-0.25-tflite-0.1.7
mobilenets-performance-128-0.25-tf-py-1.8.0
mobilenets-performance-128-0.25-tf-cpp-1.8.0
```

Reply:
Or even rename `program_id` to `lib_type`.

Reply:
Done